src/components/Calculator.js

```diff
@@ -1,24 +1,20 @@
 /* eslint-disable react/destructuring-assignment */
 /* eslint-disable react/forbid-prop-types */
```

Comment:
- [ ] Rather than disable the linter strive to fix them as this makes you more accustomed to the style guide of the tech.

Reply:
Revolved I have numbers array to calculator.js